### ISSUE TITLE
Customise mouse wheel/trackpad pad/seek/zoom behaviour

### ISSUE THREAD

**Issue by mifi** [mifi/lossless-cut#1884]
Ideas from #553 #1212 #1859:

- [x] <kbd>ctrl</kbd> (customisable) + wheel: zoom in/out
- [x] <kbd>alt</kbd> (customisable) + wheel: seek 1 frame
- [x] <kbd>shift</kbd> (customisable) + wheel: seek keyframe
- [ ] <kbd>modifier</kbd> + wheel: seek interval X seconds (customisable?)
- [ ] <kbd>modifier</kbd> + wheel: seek interval Y seconds (customisable? how many variants?)
- [ ] UI button/toggle to switch between variants https://github.com/mifi/lossless-cut/issues/1859#issuecomment-1924943827

Problem: It might be hard to map a "wheel" event to a single consistent unit (see https://github.com/mifi/lossless-cut/issues/1859#issuecomment-1932385284)

Alternatively (easier) just allow configuring multiple intervals with different modifiers.

**Comment by kulkatt** [comment]
yup, since the app encourages using keyframes as cutting points, the default behavior of mouse wheel should be panning between keyframes
